docs/configure/triggers.md

```diff
@@ -245,6 +402,7 @@ def trigger():
         "Max-Time-Received": data.get('max_time_received', 'no value'),
         "Data-Type": data.get('data_type', 'no value'),
         "File-Id": data.get('file_id', 'no value'),
+        "Trigger-Condition": data.get("trigger_condition", 'no value'),
```

Comment:
is this accurate? @RoxyFarhad and the below

Reply:
@sguequierre please actually test this

Reply:
You can use the Flask code on https://docs-test.viam.dev/3527/configure/triggers/ alongside `ngrok`. Install ngrok, which allows you to make your localhost web app accessible to the internet temporarily. You can start the "tunnel" with `ngrok http 8080` if the webapp is running on port 8080. Then the link it gives you you can put in the webhooks field

Reply:
I did just test this and it does work.

Reply:
thank you so much!